code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -589,8 +594,18 @@ var/list/ai_verbs_hidden = list( // For why this exists, refer to https://xkcd.c
 					holo_icon = getHologramIcon(icon('icons/mob/AI.dmi',"holo1"))
 				if("floating face")
 					holo_icon = getHologramIcon(icon('icons/mob/AI.dmi',"holo2"))
+				if("singularity")
+					holo_icon = getHologramIcon(icon('icons/obj/singularity.dmi',"singularity_s1"))
+				if("drone")
+					holo_icon = getHologramIcon(icon('icons/mob/animal.dmi',"drone0"))
 				if("carp")
 					holo_icon = getHologramIcon(icon('icons/mob/AI.dmi',"holo4"))
+				if("spider")
+					holo_icon = getHologramIcon(icon('icons/mob/animal.dmi',"nurse"))
+				if("bear")
+					holo_icon = getHologramIcon(icon('icons/mob/animal.dmi',"brownbear"))
+				if("slime")
+					holo_icon = getHologramIcon(icon('icons/mob/slimes.dmi',"cerulean adult slime"))

```

Comment:
Doesn't the need to update multiple locations, with the risk of typos and other copy-paste issues, to add a new hologram not make anyone else feel ill?